openpower/configs/hostboot/witherspoon.config

```diff
@@ -52,15 +52,15 @@ unset PNOR_TWO_SIDE_SUPPORT
 
 unset BMC_BT_LPC_IPMI
 
-# Enable Checktop Analysis
+# Disable Checktop Analysis
 unset ENABLE_CHECKSTOP_ANALYSIS
 unset IPLTIME_CHECKSTOP_ANALYSIS
 
-# Hostboot will detect hardware changes
+# Hostboot will not detect hardware changes
 unset HOST_HCDB_SUPPORT
 
 # set for trace debug to console
-unset CONSOLE_OUTPUT_TRACE
+set CONSOLE_OUTPUT_TRACE
```

Comment:
Separate patch?

Is this required/how people are currently booting? Or is it just a convenience for hostboot development?